rules/falco_rules.yaml

```diff
@@ -1838,18 +1838,31 @@
           registry.access.redhat.com/sematext/agent,
           registry.access.redhat.com/sematext/logagent]
 
+# Falco containers
+- list: falco_containers
```

Comment:
@jasondellaluce are the lists ok with this syntax or we would prefer to use the same syntax we have for the other lists?

Reply:
@darryk10 Are you thinking `containers` vs `images`?

I followed the example of `- list: ibm_cloud_containers` but there seems to be others `- list: redhat_io_images_privileged`.